.php_cs.dist

```diff
@@ -19,6 +19,7 @@ return PhpCsFixer\Config::create()
         'self_accessor' => false,
         // Part of @Symfony:risky in PHP-CS-Fixer 2.13.0. To be removed from the config file once upgrading
         'native_function_invocation' => array('include' => array('@compiler_optimized'), 'scope' => 'namespaced'),
+        'phpdoc_types_order' => array('null_adjustment' => 'always_last', 'sort_algorithm' => 'none'),
```

Comment:
Note that the comment above is misleading: the rule isn't part of any ruleset for now. Do you mind submitting a PR on PHP CS Fixer to update the `Symfony` ruleset?

Reply:
@julienfalque What branch in PHP-CS-Fixer should I send? :)

Reply:
Rulesets updates are done on minor releases so you should target branch master.

Reply:
@julienfalque Cool. Gonna update the comment here for now, as well, submit a patch for the ruleset in php-cs-fixer :blush: